packages/@coorpacademy-components/src/atom/icon/index.js

```diff
@@ -32,6 +33,13 @@ const SIZE_CONFIGS = {
   }
 };
 
+export const createGradientBackground = (baseColor) => {
```

Comment:
je pense que tu peux utiliser convert pour faire ton gradient (pas besoin de hextohsl), surtout que iconColor n'est toujours forcement en Hex, 
Tu peux tester ca, si ca marche, ca serait une bonne idée que tu mettes un peu de tests unitaires pour ta fonction où tu testes avec diff scénarios de iconColor (Hex, hsl et rgba)
```suggestion
export const createGradientBackground = baseColor => {
  const startColor = convert(`color(${baseColor} lightness(+59%))`);
  const endColor = convert(`color(${baseColor} lightness(+49%) saturation(+1%))`);

  return `linear-gradient(180deg, ${startColor} 0%, ${endColor} 100%)`;
};
```